editor/src/components/canvas/controls/select-mode/absolute-resize-control.tsx

```diff
@@ -234,13 +238,30 @@ const ResizeEdge = React.memo(
       [maybeClearHighlightsOnHoverEnd],
     )
 
+    const onEdgeDblClick = React.useCallback(
+      (event: React.MouseEvent<HTMLDivElement>) => {
+        if (event.detail != 2) {
+          return
+        }
+
+        runStrategies(
+          dispatch,
+          metadataRef.current,
+          selectedElementsRef.current,
+          setPropHugStrategies(invert(props.direction)),
+        )
+      },
+      [dispatch, metadataRef, props.direction, selectedElementsRef],
+    )
+
     const lineSize = ResizeMouseAreaSize / scale
     const width = props.direction === 'horizontal' ? undefined : lineSize
     const height = props.direction === 'vertical' ? undefined : lineSize
     const offsetLeft = props.direction === 'horizontal' ? `0px` : `${-lineSize / 2}px`
     const offsetTop = props.direction === 'vertical' ? `0px` : `${-lineSize / 2}px`
     return (
       <div
+        onClick={onEdgeDblClick}
```

Comment:
Can't you just use `onDoubleClick`? And then there is no need to check the event.detail